packages/manager/apps/web/client/app/domain/zone/record/domain-zone-record.controller.js

```diff
@@ -771,7 +773,7 @@ angular.module('App').controller(
     }
 
     static getSPFIncludeName() {
-      return `mx.ovh.${Environment.getRegion() === 'CA' ? 'ca' : 'com'}`;
+      return `mx.ovh.${this.coreConfig.isRegion('CA') ? 'ca' : 'com'}`;
```

Comment:
The method won't be static anymore as you're using `this` variable.

Reply:
Fixed (usage in `packages/manager/apps/web/client/app/domain/zone/record/SPF/domain-zone-record-SPF.html` updated), thanks.